src/lib.rs

```diff
@@ -81,7 +81,6 @@
 /// Unwrap a [`Poll`](std::task::Poll) value and return from the enclosing function if it was [`Pending`](std::task::Poll::Pending).
 ///
 /// This is like `try!()`, but for [`Poll`](std::task::Poll).
-#[allow(unused_macros)]
```

Comment:
Similarly here.

Reply:
I think there was a time when it was conditionally used only when a transport feature was enabled. But now it appears to be used anyway.